code/Gamemodes/Modes/FFA/FreeForAll.cs

```diff
@@ -54,6 +54,11 @@ public override string GetGameStateLabel()
 	/// </summary>
 	public Task NextTurnTask { get; set; }
 
+	public override float GetTimeRemaining()

```

Comment:
@trundler-dev Is this time remaining until next turn? Or time remaining for the entire gamemode?

Reply:
This is remaining time for the next turn. We can rename this later once we add a time limit covering the entire game.